config/test.js

```diff
@@ -3,6 +3,9 @@ var path = require('path');
 module.exports = {
   testMode: true,
   createQueue: false,
+  queue: {
```

Comment:
This will be removed once the queue is deployed.  Only for local testing with the queue rewrite.